docs/migrating11to17.md

```diff
@@ -45,5 +45,7 @@ The following table lists the new OpenJ9 features and notable changes with the O
 
 | Features and changes  | OpenJ9 release|
 |-----------------------|---------------|
-|Linux builds for all platforms use gcc 10.3 instead of gcc 7.5. See the list of [build environments](openj9_support.md#build-environments).| 0.33.0  |
-|The default operating system stack size on x64 platforms is increased from 256 KB to 512 KB to accommodate vector support. You can change the operating system stack size by using the [-Xmso](xmso.md) option.| 0.33.0   |
+|The default operating system stack size on x64 platforms is increased from 256 KB to 512 KB to accommodate vector support. You can change the operating system stack size by using the [-Xmso](xmso.md) option.|  0.33.0   |
+|On z/OS, when the VM loads an application native library (DLL), it searches for the library in the path that is specified in the `-Djava.library.path` system property or the `LIBPATH` environment variable. In Java 11 and earlier versions, the VM searched for the application native libraries by using both the `-Djava.library.path` system property and the `LIBPATH` environment variable. From Java 17 onwards, the search is based on only the `-Djava.library.path` system property value that is obtained when the VM starts. Changing the `-Djava.library.path` system property value after VM startup doesn't change the search. If the path is set on the command line by using the `-Djava.library.path` system property, only this path is used for the search. The path specified by the `LIBPATH` variable is not used to search for the application native libraries. The default value of the  `-Djava.library.path` system property is still derived from the `LIBPATH` environment variable.  |  0.47.0  |
```

Comment:
The change to only look at the `java.library.path` is not specific to z/OS, it applies to all platforms. `LIBPATH` is specific to z/OS and AIX, Linux uses LD_LIBRARY_PATH, Windows uses PATH.